samples/dynamodb/lambda/DynamoDB_1/Sales/OrderProcessor.cs

```diff
@@ -0,0 +1,44 @@
+using System.Threading;
+using System.Threading.Tasks;
+
+using Amazon.Lambda.Core;
+using Amazon.Lambda.SQSEvents;
+
+using NServiceBus;
+
+public class OrderProcessor
+{
+  #region EndpointSetup
+
+  static readonly AwsLambdaSQSEndpoint endpoint = new AwsLambdaSQSEndpoint(context =>
+  {
+    var endpointConfiguration = new AwsLambdaSQSEndpointConfiguration("Samples.DynamoDB.Lambda.Sales");
+
+
+    var advanced = endpointConfiguration.AdvancedConfiguration;
+    advanced.SendFailedMessagesTo("Samples.DynamoDB.Lambda.Error");
+    advanced.EnableInstallers();
+
+    var persistence = advanced.UsePersistence<DynamoPersistence>();
+
+    persistence.UseSharedTable(new TableConfiguration()
+    {
+      TableName = "Samples.DynamoDB.Lambda",
+    });
+
+    return endpointConfiguration;
+
+  });
+
+  #endregion
+
+  #region FunctionHandler
+
+  public async Task ProcessOrder(SQSEvent eventData, ILambdaContext context)
+  {
+    context.Logger.Log("ProcessOrder was called");
```

Comment:
is ProcessOrder the right name given this endpoint handles different message types?

Reply:
thinking more about this, I think this is good enough to leave as is.